daemon/main.c

```diff
@@ -567,7 +566,11 @@ static void get_netdata_configured_variables() {
         error("Invalid multidb disk space %d given. Defaulting to %d.", default_multidb_disk_quota_mb, default_rrdeng_disk_quota_mb);
         default_multidb_disk_quota_mb = default_rrdeng_disk_quota_mb;
     }
-
+#else
+    if (default_rrd_memory_mode == RRD_MEMORY_MODE_DBENGINE) {
+       error_report("RRD_MEMORY_MODE_DBENGINE is not supported in this platform. The agent will use memory mode ram instead.");
```

Comment:
Maybe better is `on this platform` ?

Reply:
Also we might consider moving `open_all_log_files()` earlier during startup. This message (along with others and from timezone stuff) won't make it in ?

Reply:
Copied the message as it appeared in rrdhost.c :smile:

Reply:
We can investigate what it would mean to open the files earlier to allow for the messages to be written properly in another issue

Reply:
> Copied the message as it appeared in rrdhost.c

We have needed a grammar and spelling check covering all of our log messages for some time now...